homeassistant/components/volvooncall/config_flow.py

```diff
@@ -76,10 +83,16 @@ async def async_step_user(
                 vol.Required(CONF_REGION, default=defaults[CONF_REGION]): vol.In(
                     {"na": "North America", "cn": "China", None: "Rest of world"}
                 ),
-                vol.Optional(CONF_MUTABLE, default=defaults[CONF_MUTABLE]): bool,
                 vol.Optional(
-                    CONF_SCANDINAVIAN_MILES, default=defaults[CONF_SCANDINAVIAN_MILES]
-                ): bool,
+                    CONF_UNIT_SYSTEM, default=defaults[CONF_UNIT_SYSTEM]
+                ): vol.In(
+                    {
+                        CONF_UNIT_SYSTEM_METRIC: "Metric",
```

Comment:
We don't use `CONF_` constants as config values, only as config keys. Please define new constants in the integration for these possible values.